File: code-recipes/java/src/test/java/easyship/CreateScheduledPackageRecipeTest.java
package easyship;

import util.RecipeTest;

import java.util.List;

public class CreateScheduledPackageRecipeTest extends RecipeTest {

    protected CreateScheduledPackageRecipeTest() {
        super(
                new CreateScheduledPackageRecipe(),
                List.of(
                    "easyship-createScheduledPackage"
                )
        );
    }
}

File: code-recipes/php/test/easyship/CreateScheduledPackageRecipeTest.php
<?php

namespace Test\easyship;

use Src\easyship\CreateScheduledPackageRecipe;
use Test\util\RecipeTestCase;

class CreateScheduledPackageRecipeTest extends RecipeTestCase
{
    protected function setUp(): void
    {
        $this->recipe = new CreateScheduledPackageRecipe();
        $this->responses = [
            "easyship-createScheduledPackage"
        ];
    }
}
